index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
```

Comment:
- Nicely done ❤️

Reply:
> Thank you. Does that mean my project has been approved?

Reply:
Hello @posh101, not yet... please check the comment I left under the review, once you will implement that one change, your project will be 💯  ready for approval